br/pkg/lightning/config/config.go

```diff
@@ -1185,7 +1188,7 @@ func (cfg *Config) CheckAndAdjustSecurity() error {
 			len(cfg.TiDB.Security.CertPath) > 0 || len(cfg.TiDB.Security.CertBytes) > 0 ||
 			len(cfg.TiDB.Security.KeyPath) > 0 || len(cfg.TiDB.Security.KeyBytes) > 0 {
 			if cfg.TiDB.Security.TLSConfigName == "" {
-				cfg.TiDB.Security.TLSConfigName = uuid.NewString() // adjust this the default value
+				cfg.TiDB.Security.TLSConfigName = uuid.New().String() // adjust this the default value
```

Comment:
is this needed?

Reply:
I don't understand why when I use uuid.NewString() it will report an error...